nav2_costmap_2d/test/unit/array_parser_test.cpp

```diff
@@ -38,9 +38,9 @@ TEST(array_parser, basic_operation)
   std::string error;
   std::vector<std::vector<float>> vvf;
   vvf = nav2_costmap_2d::parseVVF("[[1, 2.2], [.3, -4e4]]", error);
-  EXPECT_EQ(2, vvf.size() );
-  EXPECT_EQ(2, vvf[0].size() );
-  EXPECT_EQ(2, vvf[1].size() );
+  EXPECT_EQ((unsigned int)2, vvf.size() );
```

Comment:
```suggestion
  EXPECT_EQ(2u, vvf.size() );
```

Same for the other ones below